product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/pgbouncer_configuring.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Configuring EDB PgBouncer for BigAnimal
+navTitle: Configuring EDB PgBouncer
+---
+
+[EDB PgBouncer](/pgbouncer/latest/) is a lightweight connection pooling utility that you can use to more quickly connect to your BigAnimal cluster. It saves you time by pooling previously established connections to your server. Instead of connecting directly to your server, EDB PgBouncer negotiates your connections for you. 
+
+Perform these steps to configure EDB PgBouncer with your BigAnimal cluster.
+
+## Update the EDB PgBouncer configuration file
+
+To install EDB PgBouncer, see [Installing EDB PgBouncer](/pgbouncer/latest/01_installation/).
+
+Navigate to your BigAnimal cluster connection information: 
+
+1. Open BigAnimal. 
+
+1. Go to **Clusters > Connect**. 
+
+1. Under **Connection Info**, find your cluster connection information. The following table is an example of connection information: 
+
+   | Information field | Value |
+   |-------------------|-------|
+   | Service URI | postgres://p-zdj5rioiy1.brcxzr08qr7rbei1.enterprisedb.network:5432/edb_admin?sslmode=require |
+   | Dbname | edb_admin |
+   | Host | p-zdj5rioiy1.brcxzr08qr7rbei1.enterprisedb.network |
+   | Port | 5432 |
+   | User | edb_admin |
+
+Configure EDB PgBouncer according to the connection information:
+
+1. Update the PgBouncer configuration file `/etc/pgbouncer/pgbouncer.ini`:
+
+   ```ini
+   [databases]
+   [database name] = host=<Host>
+   port=<Port> 
+   user=<User> 
+   dbname=<Dbname>
+   [pgbouncer]
+   # Enter * to listen on all addresses:
+   listen_addr=*
+   # Enter the port to listen on:
+   listen_port=
+   # Enter your setting for authenticating users:
+   auth_type=
+   # Enter the file that contains the list of usernames and passwords:
+   auth_file= 
+   ```
+For more information, see the [PgBouncer configuration file page](https://www.pgbouncer.org/config.html). 
```

Comment:
We should probably link to our topic, which in turn links to the community topic: https://www.enterprisedb.com/docs/pgbouncer/latest/02_configuration_and_usage/